commit 288e998a415af65109a0cdbe3a8e1b3b28ab01d2 (refs/changes/10/4510/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-07-16T16:31:45+02:00 (1 year, 3 months ago)
    
    TRITON-589 TRITON-558 verbosity breaks some sdcadm tests
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/lib/procedures/index.js b/lib/procedures/index.js
index e9486cf..948416d 100644
--- a/lib/procedures/index.js
+++ b/lib/procedures/index.js
@@ -131,7 +131,6 @@ function coordinatePlan(opts, cb) {
          * If SAPI isn't in full mode, we'll error out and stop the updates
          */
         function ensureSapiFullMode(_, next) {
-            progress('Verifying SAPI full mode');
             sdcadm.sapi.getMode(function getModeCb(err, mode) {
                 if (err) {
                     next(err);
